docs/HACKING.md

```diff
@@ -198,15 +198,16 @@ like to install into the VM (e.g. bubblewrap, libsolv, etc...).
 
 A common case is testing changes in the `ostree-rs-ext` crate. Once you have
 your changes ready in a clone of the `ostree-rs-ext` repository, you can edit
-rpm-ostree's `Cargo.toml` to point to it. For example:
+rpm-ostree's `Cargo.toml` to point to it.
 
 ```
-$ sed -i "s/ostree-ext = \".*\"/ostree-ext = { path = '..\/ostree-rs-ext\/lib\/' }/" Cargo.toml
+...
+[patch.crates-io]
+ostree-ext = { path= '../ostree-rs-ext/lib/' }
```

Comment:
I had to look up what crates were (I'm not familiar with rust yet). Under the link you provided `path` uses the `"` in here instead of the single quotes, but I'm pretty sure this will not impact the code execution - right?